lib/ansible/modules/network/nxos/nxos_vrf.py

```diff
@@ -60,14 +63,27 @@
     required: false
     default: null
     version_added: "2.2"
-  route_distinguisher:
+  rd:
     description:
       - VPN Route Distinguisher (RD). Valid values are a string in
         one of the route-distinguisher formats (ASN2:NN, ASN4:NN, or
         IPV4:NN); the keyword 'auto', or the keyword 'default'.
     required: false
     default: null
-    version_added: "2.2"
+    version_added: "2.5"
```

Comment:
Is this really a new parameter?

Reply:
@Qalthos it turned out that the argument `route_distinguisher` is documented instead of `rd` but the argspec only had `rd`, not `route_distinguisher` in the module code. So I renamed the argument name in the doc. this is why the version needs to be changed to `2.5`. I don't think that shippable will not fail if I don't change the version here since the arg is renamed in the doc.

Reply:
The Shippable failure will go away after merge

Reply:
reverted the change as suggested by @gundalow